tools/netboxr/.shed.yml

```diff
@@ -0,0 +1,7 @@
+description: netboxr enables automated discovery of biological process modules by network analysis
+long_description: |
+  The NetBox algorithm identifies candidate cancer-related functional modules using a data-driven, network-based approach that combines prior knowledge with a network clustering algorithm. The network modules are derived de novo, and can be used to identify new functional gene groups that cross the boundaries of curated gene sets.
+name: netboxr
+owner: sajo25
```

Comment:
bgruening - its the TS account.

Reply:
Do you mean that I should write "bgruening" under "owner"?

Reply:
yes, sorry. this repo submits everything as bgruening. You can also submit your tool to https://github.com/galaxyproject/tools-iuc and then the owner is IUC.